bullet_train/lib/bullet_train/resolver.rb

```diff
@@ -251,10 +251,47 @@ def locale_path
       nil
     end
 
+    # In this search, we prioritize files in local themes
+    # and then look in theme gems if nothing is found.
+    def js_or_stylesheet_path
+      file_name = @needle.split("/").last
+
+      # Prioritize the current theme and fall back to
+      # the default `light` theme if nothing is found locally.
+      puts "Searching under current theme: #{current_theme.blue}"
+
+      asset_globs = [
+        "*.js", # Include JavaScript files under the app's root directory.
+        "app/assets/javascript/**/*.#{current_theme}.js",
+        "app/assets/javascript/#{current_theme}/**/*.js",
+        "app/assets/stylesheets/**/*.#{current_theme}.css",
+        "app/assets/stylesheets/#{current_theme}/**/*.css",
+      ]
+
+      files = Dir.glob(asset_globs).reject { |file| file.match?("/builds/") }
+      absolute_file_path = files.find { |file| file.end_with?(file_name) }
```

Comment:
@gazayas I went ahead and applied the change to `end_with?` so we're symmetrical with the `end_with?` call on line 283.